docs/docs/workspace-settings/README.md

```diff
@@ -0,0 +1,93 @@
+# Workspace Settings
+
+When you sign up for a Pipedream account, you'll be guided to create a workspace. After you create your workspace, your workflows and projects will be assigned to your private workspace.
+
+A workspace is a collection of **projects**, **folders** and **workflows**. You can develop workflows within a private workspace, or you can invite others to join your workspace.
+
+You can be a member of one or more workspaces.
+
+## Creating a new workspace
+
+To create another workspace, open the dropdown menu in the top left of the Pipedream dashboard.
+
+Then select **Create new workspace**.
+
+You'll be prompted to name the workspace, but don't worry you can [change the name later](/workspaces/#renaming-a-workspace).
+
+::: tip
+
+**Free** and **Professional** tier accounts are limited to owning one workspace. [Browse plans](/pricing/) that include higher workspace usage.
+
+:::
+
+
+## Switching between workspaces
+
+To switch between workspaces, open the dropdown menu in the top left of the Pipedream dashboard.
+
+Select which workspace you'd like to start working within, and your Pipedream dashboard context will change to that workspace.
+
+
+![]()
+
+## Workspace settings
+
+Find your current workspace settings like current members, under the **Settings** navigation menu item on the left hand side. Here is where you can manage your workspace settings, including the workspace name, members and member permissions.
+
+### Inviting others to a join a workspace
+
+After opening your workspace settings, open the **Membership** tab.
+
+Invite members to your workspace by entering in their Pipedream username or email address and then clicking **Send**.
```

Comment:
Usernames are going away completely. Can you update any references you find to usernames as a part of the docs change? Please sync with Pravin on the rollout, since I'm not sure exactly when that will land.